src/cryptoadvance/specter/node.py

```diff
@@ -620,13 +655,8 @@ def node_type(self):
         return "BTC"
 
     @property
-    def datadir(self):
```

Comment:
Removing the property here: Are you sure you checked for all references of this? Check e.g. `specter.py:289` and maybe in more occasions?!

Reply:
Outdated, I introduced the property and removed it again. Not sure why GitHub showed it at all.